### ISSUE TITLE
Internal failure when creating a subscription with multiple fields

### ISSUE THREAD

**Issue by tylerhou** [rmosolgo/graphql-ruby#1813]
When trying to create a subscription on multiple fields:

```graphql
subscription {
  first_field {
    ...
  }
  second_field {
    ...
  }
}
```

I get an error on this line:

```ruby
module GraphQL
  module Pro
    class Subscriptions < GraphQL
      class RedisStorage
        ...
        def write_subscription(query, topics)
          sub_id = query.context[:subscription_id] || raise("Failed to get subscription id")
          @redis.sadd(all_topics_key, *topics) # << ERROR
          topics.each do |topic|
```
.

[This is because redis#sadd is not variadic](https://www.rubydoc.info/github/redis/redis-rb/Redis:sadd); instead it accepts a string or an array of strings as the second argument.

The fix is to remove the splat operator:

```ruby
          @redis.sadd(all_topics_key, topics)
```

I've tested this fix on GraphQL Pro 1.7.11 and it works.

**Comment by rmosolgo** [comment]
Thanks for the details on this issue and for the fix, I'll cut a release soon!

**Comment by tylerhou** [comment]
Thank you for your work as well!

**Comment by rmosolgo** [comment]
I'm so sorry this slipped off my radar! I've (finally) released graphql-pro 1.10.7 which includes the fix you posted here. Thanks for sharing that solution ... and for your patience! Please let me know if you run into any other trouble.
